src/pages/PageHome.vue

```diff
@@ -11,81 +11,28 @@
-        /></RouterLink>
+        <RouterLink :to="{ name: 'Orgchart' }" class="button">
+          <span>View the Org Chart</span>
+          <Icon id="chevron-right" :width="24" :height="24" />
+        </RouterLink>
       </p>
     </div>
     <CardRow>
-      <Card>
-        <div class="card__icon">
-          <Icon id="irc" :width="64" :height="64"></Icon>
-        </div>
-        <h2>
-          <a
-            href="https://discourse.mozilla.org/c/iam/dinopark"
-            target="_blank"
-            rel="noopener noreferrer"
-            >Your Feedback Matters</a
-          >
-        </h2>
-        <p>
-          Join us on
-          <a
-            href="https://discourse.mozilla.org/c/iam/dinopark"
-            target="_blank"
-            rel="noopener noreferrer"
-            >Discourse</a
-          >
-          and let us know how we can make the Mozilla Directory even better.
-        </p>
-      </Card>
-      <Card>
-        <div class="card__icon">
-          <Icon id="lock" :width="64" :height="64"></Icon>
-        </div>
-        <h2>
-          <a
-            href="https://discourse.mozilla.org/t/dinopark-faq/33564"
-            target="_blank"
-            rel="noopener noreferrer"
-            >Your Privacy is Protected</a
-          >
-        </h2>
-        <p>
-          Your Staff profile data is only visible to Staff and NDA’d people by
-          default. You can edit this via your profile privacy settings. Read our
-          <a
-            href="https://discourse.mozilla.org/t/dinopark-faq/33564"
-            target="_blank"
-            rel="noopener noreferrer"
-            >Discourse post</a
-          >
-          for details.
-        </p>
-      </Card>
-      <Card>
-        <div class="card__icon">
-          <Icon id="edit" :width="64" :height="64"></Icon>
-        </div>
-        <h2>
-          <a
-            href="https://discourse.mozilla.org/t/edit-mode-announcement/38563"
-            target="_blank"
-            rel="noopener noreferrer"
-            >Recent Additions</a
-          >
-        </h2>
-        <p>
-          Profile editing is now live with privacy controls and synchronized
-          Workday information. Read more about this and improved people search
-          capabilities in our
-          <a
-            href="https://discourse.mozilla.org/t/edit-mode-announcement/38563"
-            target="_blank"
-            >Discourse post</a
-          >.
-        </p>
+      <Card v-for="(card, idx) in cards" :key="idx">
+        <a
+          v-bind:href="card.link"
+          target="_blank"
+          rel="noopener noreferrer"
+          class="card__link"
+        >
+          <div class="card__icon">
+            <Icon v-bind:id="card.icon" :width="64" :height="64"></Icon>
+          </div>
+          <div class="card__content">
+            <h2>{{ card.headline }}</h2>
+            <p v-html="card.description"></p>
```

Comment:
We produce nested anchors here, that's not w3c conform. I'm really not sure it there's a better way than Hidde did it with the pseudo element.